contracts/MedianOracle.sol

```diff
@@ -116,7 +120,7 @@ contract MedianOracle is OwnableUpgradeable, IOracle {
 
     /**
      * @notice Pushes a report for the calling provider.
-     * @param payload is expected to be 18 decimal fixed point number.
+     * @param payload is expected to be a fixed point number with 18 decimals.
```

Comment:
```suggestion
     * @param payload is expected to be a fixed point number with `DECIMALS` decimals.
```

Is this worth it or just more confusing?
https://docs.soliditylang.org/en/latest/natspec-format.html#dynamic-expressions